runtime/compiler/optimizer/InlinerTempForJ9.cpp

```diff
@@ -286,7 +286,7 @@ TR_J9InlinerPolicy::isMethodInSharedCache(TR_ResolvedMethod *interfaceMethod)
    TR_J9VMBase *fej9 = (TR_J9VMBase *)(comp()->fe());
    TR_OpaqueClassBlock *clazz = interfaceMethod->classOfMethod();
    void *dummy;
-   bool clazzInCache = fej9->sharedCache()->isPointerInSharedCache((void*)fej9->getPersistentClassPointerFromClassPointer(clazz), dummy);
+   bool clazzInCache = fej9->isPointerInSharedCache((void*)fej9->getPersistentClassPointerFromClassPointer(clazz), dummy);
```

Comment:
I don't see a single use of `TR_J9InlinerPolicy::isMethodInSharedCache(TR_ResolvedMethod *interfaceMethod)` anywhere in the code, so you could probably just delete this method entirely.